apps/user-profile/src/components/modules/UserOnboardingModal/StyledUserOnboardingModal.js

```diff
@@ -8,11 +9,35 @@ export const ModalContainer = styled.div`
 
 export const ModalHeader = styled.div`
   width: 100%;
+  display: flex;
+  align-items: center;
+  background-color: ${theme.colors.Black};
+  color: #F0EDEE;
```

Comment:
This is the color present in the Figma design and in packages\UI\src\styles\theme.ts
I wasn't able to use ${theme.color.GREYSCALE_OFF_WHITE} to get that because I am using the theme in the file path import theme from "../../../styles/theme";
I have tried using import {theme as theme1} from ... and import {theme as theme1} from ... but it was not working. that is why I am not using the build in colour from the theme file here

Reply:
THis is fine, i will be configuring the styling system for user profile project later on

Reply:
@HavreLoic theme can used as follows
```
${({ theme }) =>theme.color.GREYSCALE_OFF_WHITE };
```